From 002ab63145afe73d0231e599c1f8afc011193f96 Mon Sep 17 00:00:00 2001
From: Vursc <vursc@vursc.org>
Date: Thu, 7 Nov 2024 15:13:56 +0000
Subject: [PATCH] hwdb: fix broken numpad paren keys on Lenovo Thinkbook 16 G6+
 2024

(cherry picked from commit eb03dffd9794ac7ebfa1995ca9fdf72987b05535)

Resolves: RHEL-49810
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index b8bd7f0fb5..236a145dc5 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1149,6 +1149,11 @@ evdev:name:SIPODEV Lenovo HID Device:dmi:*:svnLENOVO:*:pvrLenovoideapadD330-10IG
 evdev:name:SIPODEV Lenovo HID Device Consumer Control:dmi:*:svnLENOVO:*:pvrLenovoideapadD330-10IGM:*
  KEYBOARD_KEY_c00ff=fn_esc                              # Fn+Tab (FnLk toggle)
 
+# Lenovo Thinkbook 16 G6+ 2024
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnLENOVO:pn21LG:pvr*
+ KEYBOARD_KEY_0a=!9
+ KEYBOARD_KEY_0b=!0
+
 ###########################################################
 # LG
 ###########################################################
